tests/test_models_vae.py

```diff
@@ -96,7 +96,7 @@ def test_output_pretrained(self):
             model.config.in_channels,
             model.config.sample_size,
             model.config.sample_size,
-            generator=torch.manual_seed(0),
+            generator=generator,
```

Comment:
I think this was the original intention. Unfortunately it doesn't fix the failing test.

Reply:
@anton-l I think this is the reason why some tests are currently failing on master - could you take a look?